packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -232,9 +235,13 @@ export class EmberishCurlyComponent extends GlimmerObject {
   public attrs: Attrs;
   public element: Element;
   public parentView: Component = null;
+  public args: ProcessedArgs;
+  public argsRevision: number;
 
   static create(args: { attrs: Attrs }): EmberishCurlyComponent {
-    return super.create(args) as EmberishCurlyComponent;
+    let result = super.create(args) as EmberishCurlyComponent;
+    result.argsRevision = result.args && result.args.tag && result.args.tag.value();
```

Comment:
Doh. I meant to use it inside of the component update but we don't really have to since we compute the value for each update (unlike in Ember where we only recompile if the tag revision changes). Will remove.